### ISSUE TITLE
[BUG] Interval Set Too High

### ISSUE THREAD

**Issue by SuspishFish** [ZsoltGajdacs/BlinkReminder#22]
I only need short breaks in BlinkReminder, I didn't have a need for long breaks. Since there doesn't seem to be a way to turn off long breaks, I set "Interval of long breaks" to something comedically large, like 999999.0 minutes. That was a bad idea. Now BlinkReminder closes itself and every time I attempt to open it. I'm presuming setting the interval this high causes it to crash. BlinkReminder closes before there's any opportunity to change the settings.

I've tried restarting the computer. Same issue happens, it closes immediately after it's opened. I've tried uninstalling and reinstalling BlinkReminder. It still closes immediately after opening. Testing it on another computer, the settings are saved between reinstalls.

Is there a way to access the settings outside of having the program open? Or a way to delete the settings and to do a fresh reinstall?

I'm using a Windows 10 OS. BlinkReminder version 0.8.0.

-Mark

![settings](https://user-images.githubusercontent.com/91575307/135181100-7abe1596-c5fa-40ea-b0db-1f9f4c3ed550.png)

**Comment by ZsoltGajdacs** [comment]
Wow. I didn't think anyone was still using this app.

The settings path is: c:/users/{username}/appdata/roaming/blinkreminder.

Delete the whole directory.

**Comment by ZsoltGajdacs** [comment]
As for the bug itself, I've been planning on reorganizing the app, but don't have much time. I started migrating it to .net5, so I'll handle it along the way.

**Comment by SuspishFish** [comment]
That is just what I needed. Thank you!

I was having issues with strained eyes which I realized was because I wasn't blinking enough while working on a screen. I struggle to remember to blink as I'm working so I looked up an app to remind me and yours was towards the top of the search. I appreciate the simple layout it has.

I wasn't sure anyone was still monitoring this app so I wasn't sure I'd receive a response. Thank you so much for taking the time to respond! Hope you have a great day.
